libcloud/ovh.cfg.orig

```diff
@@ -70,12 +70,14 @@ libcloud:
             - systemctl restart ntpd.service
       driver: openstack
       driver_args:  # driver args are passed directly to the libcloud driver
+        ex_force_service_region: 'SBG5'
+        #ex_force_auth_url: 'https://auth.cloud.ovh.net/v2.0/tokens'
```

Comment:
nit: why not just dropping the 2 lines? we have git to remember old stuff.

Reply:
I don't want to do this

Reply:
it is not old stuff, it is still working on OVH